src/Evolution/Executables/RelativisticEuler/Valencia/EvolveValencia.hpp

```diff
@@ -125,6 +125,7 @@ struct EvolutionMetavars {
 
   static constexpr bool local_time_stepping =
```

Comment:
[random line because I can't reach it] Not necessary for this PR, but @nilsdeppe do you why clang-tidy is upset about the Tag? I'm a bit confused by the Tag because assuming it's the tag in `src/PointwiseFunctions/Hydro/Tags.hpp`, it takes a `bool` and `size_t` as templates and not a type.

Reply:
I doubt that executable compiles.

Reply:
Indeed it does not. I guess we should either update it or remove it

Reply:
We would like to remove it, but finite time 🙃

Reply:
Ah yes. Ok I'll merge this then